### PR TITLE
Add ticker + chain in withdraw page

### LINKED ISSUE #1126: [WITHDRAW] Add ticker + chain to asset icon
## HAVE

Screenshot from 2021-03-17 16-38-41

## WANT

```
--------
| icon |  TICKER     amount (price)
|      |  CHAIN
--------
```

### PR DESCRIPTION
**Pull request by junkai121**
close #1126

### CODE REVIEW

File: src/renderer/components/deposit/withdraw/Withdraw.tsx [hunk 1]
```diff
@@ -325,6 +329,10 @@ export const Withdraw: React.FC<Props> = ({
 
       <Styled.AssetContainer>
         <Styled.AssetIcon asset={asset} network={network} />
+        <Styled.Label nowrap>
+          <Styled.TickerLabel className="ticker">{asset.ticker}</Styled.TickerLabel>
+          <Styled.TickerLabel className="small">{asset.chain}</Styled.TickerLabel>
```
Comment (veado) [inline src/renderer/components/deposit/withdraw/Withdraw.tsx:334]: Pls provide `Styled.TickerLabel` + `Styled.ChainLabel`. Both are very similar styled components, but we should not to override it by using `className` in other components. Imagine we change ".small" to ".chain" or so, all overridden styles will be lost.

Comment (veado) [reply src/renderer/components/deposit/withdraw/Withdraw.tsx:334]: Let do it with another PR after @sarawut11 has finished #1136. _Approving now..._

**Review by veado — APPROVED** [review]

File: src/renderer/components/deposit/withdraw/Withdraw.tsx (hunk 1 above)
Comment (junkai121) [reply src/renderer/components/deposit/withdraw/Withdraw.tsx]: Just fixed and pushed

**Review by veado — APPROVED** [review]

File: src/renderer/components/deposit/withdraw/Withdraw.tsx (hunk 1 above)
Comment (veado) [reply src/renderer/components/deposit/withdraw/Withdraw.tsx]: _It seems we can use `AssetData` here. Anyway, this change is not needed yet..._

**Pull request merged**